src/test/resources/CalculatorSuits/AllTestsTestSuit.xml

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+<suite name="All Tests">
+    <test name="All tests">
+        <classes>
+            <class name="hw1.CalculatorAddTest"></class>
+            <class name="hw1.CalculatorSubtractTest"></class>
+            <class name="hw1.CalculatorMultiplyTest"></class>
+            <class name="hw1.CalculatorDivideTest"></class>
+        </classes>
+    </test>
+</suite>
```

Comment:
Why do you decide include set of all classes to common suite? Do we have other way add all tests to the main suite?

Reply:
You can specify a package instead of classes